core/src/config.rs

```diff
@@ -103,3 +103,23 @@ fn admin_public_key_from(
 
     Ok(PublicKey::from(&SecretKey::from_bytes(&hash)?))
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
Should be `tests`: https://doc.rust-lang.org/1.0.0/style/testing/unit.html

Reply:
👍